src/components/modules/puzzles/puzzlesBanner/PuzzlesBanner.tsx

```diff
@@ -177,28 +210,40 @@ export const PuzzlesBanner: React.FC<PuzzlesBannerProps> = ({ tracking, submitCo
                                     <img src={appStore.google} alt="Get it on Google Play" />
                                 </Link>
                             </div>
-                            {isKeyboardUser && (
-                                <p css={minimiseHint}>
-                                    <SvgInfo /> You can minimise this banner using the escape key
-                                </p>
-                            )}
+                            <div css={siteMessage}>
+                                Already a subscriber?
+                                <br css={showOnDesktop} />{' '}
+                                <Link
+                                    target="_blank"
+                                    rel="noopener noreferrer"
+                                    css={signInLink}
+                                    data-link-name={signInComponentId}
+                                    onClick={onSignInClick}
```

Comment:
I think we should just use the `href` prop here rather than navigating in the event handler- the click handler will execute before navigation happens, so there's no need to do it manually.

Reply:
It'll also remove the need to use `event.preventDefault()` in the click handler- no need to prevent the default navigation.